docs/reference/edot-sdks/android/index.md

```diff
@@ -0,0 +1,78 @@
+---
+navigation_title: EDOT Android
+description: The Elastic Distribution of OpenTelemetry Android (EDOT Android) is an APM agent based on OpenTelemetry. It provides built-in tools and configurations to make the OpenTelemetry SDK work with Elastic using as little code as possible while fully leveraging the combined forces of Elasticsearch and Kibana for your Android application.
+applies_to:
+  stack:
+  serverless:
+    observability:
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-sdk
+mapped_pages:
+  - https://www.elastic.co/guide/en/apm/agent/android/current/intro.html
+  - https://www.elastic.co/guide/en/apm/agent/android/current/index.html
+---
+
+# Elastic Distribution of OpenTelemetry Android
+
+The Elastic Distribution of OpenTelemetry Android (EDOT Android) is an APM agent based on [OpenTelemetry](https://opentelemetry.io/). EDOT Android provides built-in tools and configurations to make the [OpenTelemetry SDK](https://opentelemetry.io/docs/languages/java/) work with your {{stack}} using as little code as possible.
+
+All the features provided by the [OpenTelemetry SDK](https://github.com/open-telemetry/opentelemetry-java) are available for you to use and are pre-configured by the agent to work properly out of the box. This allows you to fulfill various use cases to better understand your app's performance.
+
+## Distributed tracing
+
+Distributed tracing allows you to see the full picture of how long your application has to wait for your backend services to provide a response, and why, as shown in the following example:
+
+:::{image} images/intro/distributed-tracing.png
+:alt: Distributed tracing
+:::
+
+The previous image shows three spans from {{kib}}:
+
+- The first span is created in your application to track an HTTP request to your backend service.
+- The second span is created within your backend service and shows how long it took to receive the client request and provide a response.
+- The last span is also created within your backend service when it queries other resources internally. In this case, it makes an HTTP request to another service, but in other cases, it could involve querying a database.
+
+You can select any of the spans to view their full details if you need more contextual information from each.
+
+:::{note}
+For distributed tracing to work properly, configure your backend services to send telemetry to the {{stack}} as well.
+:::
+
+## Session review
+
+The SDK attaches session information to each span and log generated from your application. This allows you to create queries that group all the telemetry that belongs to a session and form a session event timeline. This is useful to identify the most common actions performed by your users, as well as tracing the steps leading up to errors they might encounter.
+
+## Alerts and dashboards
```

Comment:
I'd leave the original title "More" instead and would also add the extra bullet point that sends users to discover the rest of Elastic's tools. This is due to what I mentioned earlier about android devs most likely not being familiar with Elastic in general.